views/partials/announcements.html

```diff
@@ -7,7 +7,7 @@ <h3 class='underline-title'><a href="../announcepage">Announcements</a>
 					ng-if="isAuthorised('ALTER_ANNOUNCEMENTS')"></span>
 		</h3>
 		<div class='row innerrow'>
-			<module-item ng-repeat="item in announcements" item="item" type="0"></module-item>
+			<module-item ng-repeat="item in announcements | limitTo:3" item="item" type="0"></module-item>
```

Comment:
Wouldn't it be more efficient to leave the server to limit the amount of announcements returned?
That can be done in the same line with the request above by adding the count again.

Reply:
Yes and no. I thought about this as well, but it depends on the situation. If we add the parameter `count=3`, then if someone visits the overview page, and subsequently visits the announcements page, there will be 2 `GET` requests. One with `count=3` and one without. Whereas if we do not add it, there is only one `GET` request. Hence, it is slightly faster in that case. (We're talking at most a few milliseconds here.) Of course, if someone only visits the overview page, it is slightly slower.
The reason why I added it is because the controllers for announcements and generalinfo are shared among the overview and the specific pages, to avoid duplication. If I want to add the `count=3` parameter, I would have to create an extra controller with almost the exact same code.

So both choices are reasonable, in my opinion. I just went with this one. I do not have a clear preference so if you prefer it like that I could change it in that way.

Reply:
Yeah I can see the logic behind it as  well now. Both are equally good in my eyes too. I am accepting this PR.